packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -63,6 +63,25 @@ export interface InstanceProps {
    * @default - default master key
    */
   readonly performanceInsightEncryptionKey?: kms.IKey;
+
+  /**
+   * Whether to enable automatic upgrade of minor version for the DB instance.
+   *
+   * @default - true
+   */
```

Comment:
Should have an empty line after this line.

Reply:
Perhaps it's not necessary.

Reply:
Yep, I think this was referring to a different line in the original diff 🙂. Please disregard.